.github/workflows/pull_request.yaml

```diff
@@ -4,44 +4,40 @@ on:
   push:
     branches:
     - 'master'
+    - 'main'
   pull_request:
 
 jobs:
   test:
     name: Tests
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-22.04
     steps:
     - name: Cancel Previous Runs
-      uses: styfle/cancel-workflow-action@0.4.0
+      uses: styfle/cancel-workflow-action@0.11.0
       with:
         access_token: ${{ github.token }}
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
       with:
         fetch-depth: 0
-    - uses: google-github-actions/setup-gcloud@v0.2.1
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.8
+    - uses: google-github-actions/setup-gcloud@a48b55b3b0eeaf77b6e1384aab737fbefe2085ac
```

Comment:
why this pin?

Reply:
Unsure copied from main and trusted prior art

Reply:
https://github.com/solo-io/go-utils/commit/ab76a9f5217cd95d8b8c1eca87e9e7087864aba0